Mathlib/RingTheory/Ideal/QuotientOperations.lean

```diff
@@ -451,12 +447,13 @@ theorem quotient_map_comp_mkₐ {I : Ideal A} (J : Ideal B) (f : A →ₐ[R₁]
 where`J = f(I)`. -/
 def quotientEquivAlg (I : Ideal A) (J : Ideal B) (f : A ≃ₐ[R₁] B) (hIJ : J = I.map (f : A →+* B)) :
     (A ⧸ I) ≃ₐ[R₁] B ⧸ J :=
-  { quotientEquiv I J (f : A ≃+* B) hIJ with commutes' := fun r => by
-  { -- Porting note: Needed to add the below lemma because Equivs coerce weird
-    have : ∀ (e : RingEquiv (A ⧸ I) (B ⧸ J)), Equiv.toFun e.toEquiv = FunLike.coe e := fun _ ↦ rfl
-    rw [this]
-    simp only [quotientEquiv_apply, RingHom.toFun_eq_coe, quotientMap_algebraMap,
-  RingEquiv.coe_toRingHom, AlgEquiv.coe_ringEquiv, AlgEquiv.commutes, Quotient.mk_algebraMap]}}
+  { quotientEquiv I J (f : A ≃+* B) hIJ with
+    commutes' := fun r => by
+    { -- Porting note: Needed to add the below lemma because Equivs coerce weird
```

Comment:
These curly brackets are a lean3-ism, right?

Reply:
Don’t you need them with the `with` syntax?

Reply:
Oh. *Those* braces